com.github.rigellute.spotify-tui.yaml

```diff
@@ -0,0 +1,28 @@
+command: spt
+id: com.github.rigellute.spotify-tui
+separate-locales: false
+finish-args:
+  - "--share=network"
+  - "--filesystem=home"
+modules:
+  - name: spt
+    sources:
+    - sha256: 71e8e962bc9175003e01de1da4aafeab66d6dd7f40511ce32d734646a98b3e04
+      type: file
+      url: https://github.com/Rigellute/spotify-tui/releases/download/v0.13.0/spotify-tui-linux.tar.gz
```

Comment:
You should try building from source.

Reply:
I'll try to reproduce the Rust build from source.